paper-api/src/main/java/org/bukkit/EntityEffect.java

```diff
@@ -1,122 +1,136 @@
 package org.bukkit;
 
 import com.google.common.base.Preconditions;
-import org.bukkit.entity.Ageable;
-import org.bukkit.entity.ArmorStand;
-import org.bukkit.entity.Cat;
-import org.bukkit.entity.Dolphin;
-import org.bukkit.entity.Egg;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.EvokerFangs;
-import org.bukkit.entity.Firework;
-import org.bukkit.entity.Fox;
-import org.bukkit.entity.Goat;
-import org.bukkit.entity.Guardian;
-import org.bukkit.entity.Hoglin;
-import org.bukkit.entity.IronGolem;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.entity.Player;
-import org.bukkit.entity.Rabbit;
-import org.bukkit.entity.Ravager;
-import org.bukkit.entity.Sheep;
-import org.bukkit.entity.Sniffer;
-import org.bukkit.entity.Snowball;
-import org.bukkit.entity.Squid;
-import org.bukkit.entity.Tameable;
-import org.bukkit.entity.TippedArrow;
-import org.bukkit.entity.Villager;
-import org.bukkit.entity.Warden;
-import org.bukkit.entity.Witch;
-import org.bukkit.entity.Wolf;
-import org.bukkit.entity.Zoglin;
-import org.bukkit.entity.ZombieVillager;
+import io.papermc.paper.datacomponent.DataComponentTypes;
+import org.bukkit.entity.*;
```

Comment:
Please do not use wildcard imports.
They are generally considered not too stable and iirc even banned in like the google code style.
IJ sometimes does that, but you can iirc just incr the value of imports to like 99999.